Automattic/jetpack
# calypsoify: use css vars from calypso-color-schemes

related to #11478

#### Changes proposed in this Pull Request:

* Add a `PostCSS` build step to calypsoify's gulp task for processing styles
* Move SASS color variables over to use CSS custom properties (using `@automattic/calypso-color-schemes`

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?

p9oQ9f-9o-p2

#### Testing instructions:

* one part is to make sure each SASS color variable gets replaced with their correct CSS custom property replacement

test locally:

* Start your local Jetpack dev environment and make sure Calypsoify is enabled (see my diff below)

```diff
diff --git a/modules/module-extras.php b/modules/module-extras.php
index 989512dcb..a3c976fc6 100644
--- a/modules/module-extras.php
+++ b/modules/module-extras.php
@@ -14,6 +14,7 @@
  */
 $tools = array(
        // Always loaded, but only registered if theme supports it.
+       'calypsoify/class.jetpack-calypsoify.php',
        'custom-post-types/comics.php',
        'custom-post-types/testimonial.php',
        'custom-post-types/nova.php',
@@ -40,7 +41,7 @@ $tools = array(

 // Some features are only available when connected to WordPress.com.
 $connected_tools = array(
-       'calypsoify/class.jetpack-calypsoify.php',
+       /* 'calypsoify/class.jetpack-calypsoify.php', */
        'plugin-search.php',
        'simple-payments/simple-payments.php',
        'woocommerce-analytics/wp-woocommerce-analytics.php',
```

* Open http://localhost/wp-admin/plugin-install.php?calypsoify=1 and verify that all colors match what's currently used in Calypsoify on master
* Open Gutenberg and make sure all colors are in place
* Make sure the CSS custom props fallback is there so colors are available in IE11
